CODEOWNERS

```diff
@@ -54,6 +54,8 @@ build.json @home-assistant/supervisor
 /tests/components/aemet/ @Noltari
 /homeassistant/components/agent_dvr/ @ispysoftware
 /tests/components/agent_dvr/ @ispysoftware
+/homeassistant/components/aidot/ @aidotdev2024
+/tests/components/aidot/ @aidotdev2024
```

Comment:
Same here. (run hassfest after updating the manifest)

Reply:
Sorry, I made a mistake. I will make some corrections